resources/views/external-link-confirm.blade.php

```diff
@@ -13,13 +13,13 @@
             Livewire.emit('openModal', '{{ $identifier }}')
         }
     }"
-    class="inline-block items-center space-x-2 font-semibold break-all cursor-pointer link"
+    class="inline-flex items-center font-semibold break-all cursor-pointer link"
```

Comment:
doing that causes this (the icon no longer aligns with the text when it wraps)
![image](https://user-images.githubusercontent.com/35610748/126489313-516d57a5-e3bc-419c-b3b4-0d1eaaa2d227.png)